prow/cluster/hook_deployment.yaml

```diff
@@ -41,6 +41,7 @@ spec:
         - --dry-run=false
         - --github-endpoint=https://api.github.com
         - --config-path=/etc/config/config.yaml
+        - --github-token-path=/etc/github/oauth
```

Comment:
was this missing previously? is it required? our cluster is running without it

Reply:
It is of course running without - but you can see warnings in the hook logs about this soon becoming required. There are other warnings too.

Reply:
Are you sure the warnings are about this and not about not using ghproxy?

Reply:
About both

Reply:
alright then, let's keep it in. yeah, the gh-proxy was on my list for a while, never got around to doing that